go.mod

```diff
@@ -17,7 +17,7 @@ require (
 	github.com/containers/image/v5 v5.24.1
 	github.com/containers/ocicrypt v1.1.7
 	github.com/containers/psgo v1.8.0
-	github.com/containers/storage v1.45.3
+	github.com/containers/storage v1.45.4
```

Comment:
A surprising number of other dependencies are bumped with .4.

@giuseppe @nalind @rhatdan PTAL
Is everything in order?

Reply:
Yes, everything being changed in this PR's `vendor/modules.txt` corresponds to changes in made in containers/storage's `vendor/modules.txt` between v1.45.3 and v1.45.4.  New dependency update bot.

Reply:
Ouch.  v1.45.4 was not cut from the correct branch (https://github.com/containers/storage/commits/release-1.45).

There is a release branch for 1.45 but .5 was cut from main.

Can we hold this PR until there is v1.45.6 cut from the correct branch?

Reply:
@rhatdan @giuseppe PTAL